README.md

```diff
@@ -158,9 +168,606 @@ search.start();
 - If you use Vue, check out [meilisearch-vue](https://github.com/meilisearch/meilisearch-vue/)
 - If you use Angular, check out [meilisearch-angular](https://github.com/meilisearch/meilisearch-angular/)
 
-## Compatibility with MeiliSearch
 
-This package only guarantees the compatibility with the [version v0.20.0 of MeiliSearch](https://github.com/meilisearch/MeiliSearch/releases/tag/v0.20.0).
+## 🤖 Compatibility with MeiliSearch and InstantSearch
+
+This package only guarantees: compatibility with:
+
+- The version v0.20.0 of [MeiliSearch](https://github.com/meilisearch/MeiliSearch/releases/tag/v0.20.0).
```

Comment:
Can you follow the same presentation as for strapi? It will make easier the update of this part of the README with meili-bot. The sentence should be the same 😇